TelenorNBIoT.cpp

```diff
@@ -366,52 +366,47 @@ bool TelenorNBIoT::sendString(IPAddress remoteIP, const uint16_t port, String st
     return sendBytes(remoteIP, port, str.c_str(), str.length());
 }
 
-bool TelenorNBIoT::receiveFrom(char *ip, uint16_t *port, char *outbuf, uint16_t *length, uint16_t *remain)
+size_t TelenorNBIoT::receiveBytes(char *buffer, uint16_t bufferLength)
 {
+    size_t readLength;
+
     if (_socket == -1) {
-        return false;
-    }
-    if (length == NULL) {
-        return false;
+        return 0;
     }
-    sprintf(buffer, RECVFROM, _socket);
+    
+    sprintf(buffer, RECVFROM, _socket, bufferLength);
```

Comment:
`buffer` used to refer to `this.buffer`, but now refers to the `buffer` parameter.  Rename the parameter but not on this line or the next.